inox2d/Cargo.toml

```diff
@@ -22,3 +22,9 @@ clap = { version = "4.1.8", features = ["derive"] }
 
 [features]
 owo = ["dep:owo-colors"]
+[dependencies.uuid]
+version = "1.7"
+features = [
+    "v4",                # Lets you generate random UUIDs
+    "fast-rng",          # Use a faster (but still sufficiently random) RNG
+]
```

Comment:
they're cheap to clone, actually 128-bit uuids, and bring free serialization. I would suggest at least changing the existing implementation to take 64-bit unsigned integers instead of u32, as that's what most javascript runtime will actually produce as a Number

Reply:
Currently [the reference implementation has `Param.uid` as a `uint`](https://github.com/Inochi2D/inochi2d/blob/2d1a8014a1a7aa352d40a26773c34107cfb7394e/source/inochi2d/core/param/package.d#L143-L146), which in D is a signed 32-bit integer. If inochi2d gets eventually patched to use an actual UUID as a parameter ID, I'll gladly do this, but I think it's best to keep the parameter ID a `u32` for now.